docusaurus/src/components/Architecture.tsx

```diff
@@ -1,6 +1,6 @@
-import React from "react";
+import React, { VFC } from "react";
```

Comment:
Is `React` still needed here?

This comments fits also to other files in this PR. Please check them out if necessary

Reply:
I agree. My mistake.

Reply:
React import is still required otherwise html elements like section, div etc. give us an error -> 'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.ts(2686) it is the same in other files too. I keep named imports like { VFC } or { FC } in such a files for components signatures consistency with files where we do not need React import and use { VFC } or { FC } named imports only.